pages/beef.js

```diff
@@ -1,10 +1,8 @@
 import SessionWrapper from "../components/SessionWrapper";
-import SessionInput from "../components/SessionInput";
```

Comment:
Maybe worth adding a comment to these pages that they are for practical testing / demo purposes